docs/variants/dell_optiplex/recovery.md

```diff
@@ -1,85 +1,211 @@
 # Recovery
 
----
+Following documentation describes the process of recovering hardware from brick
+state with [RTE](../../../transparent-validation/rte/introduction/) and Dasharo
+open-source firmware. Following procedure is supported for following models
+
+<center>
+
+| Vendor | Model |
+:-------:|:-----:|
+|Dell    | OptiPlex 7010 SFF |
+|Dell    | OptiPlex 7010 DT |
+|Dell    | OptiPlex 9010 SFF |
 
-## **Prequisitions**
+</center>
 
-1. To proceed with recovery `.rom` file with backup vendor firmware will be
-    neccesary eg. `backup.rom`.
+## Step 1: Prepare hardware
 
-    Backup file should be generated before making any changes in device flash
-    chip according to documentation in
-    [BIOS backup](https://docs.dasharo.com/variants/dell_optiplex/initial-deployment/#bios-backup)
-    section.
+1. To prepare hardware please follow
+[hardware preparation](../initial-deployment/#hardware-preparation) procedure.
+2. Follow instruction in the video to remove heatsink:
```

Comment:
Whether all points should not be declared as `1`? If the text formatting is correct, these `1` should be converted into consecutive numbers.

Reply:
Fixed https://github.com/Dasharo/docs/pull/139/commits/1040d116a5113f6ed8ec300850b2b1b8441499f4